text/0000-informational-commands.md

```diff
@@ -0,0 +1,775 @@
+- whether it is the user's default or a project-pinned version
+- for tools, the associated runtime and packager versions
+
+## Output modes
+
+The tool will support multiple modes (two initially), which include exactly the same information but presented in different human- or machine-friendly formats. All modes include the same information for runtimes, packagers, and tools: name, version, whether it is a default or project-specified version, and (for tools) the Node version and packager (i.e. platform).
+
+- "human" mode, the default if the context is a user-facing terminal; also invokable with `--print=human` in any context. An indented listing of the user's current runtime, packager (if specified), and any installed binaries. See the detailed sections below for examples of the format.
+
+- "plain" mode, the default if the context is not a user-facing terminal (e.g. when piped into another command); also invokable with `--print=plain` in any context. A simple plain text format which prints a line per runtime, packager, or tool, with space-separated output on each line.
+
+    - runtimes:
+
+        ```
+        runtime node@<version>
+        ```
+
+    - packagers:
+
+        ```
+        packager (yarn|npm)@<version>
+        ```
+
+    - tools: 
+
+        ```
+        tool <tool name> / <package name>@<package version> [node@<version>] [<yarn|npm>@<version>] [(default|current @ <project path>)]
+        ```
+
+This RFC does not propose, but allows for the possibility of, a JSON mode (`--print=json`) or similar at a later time if that proves desirable.
+
+## Detailed command output
+
+Here we supply a worked example of each *variant* in both *output styles*.
+
+### Assumed Configuration
+
+Throughout, we will assume the user has the following configuration for illustrative purposes:
+
+- Node versions installed: v12.2.0, v11.9.0, v10.15.3 (default), v8.16.0
+- Yarn versions installed: v1.16.0, v1.12.3 (default)
+- Tools installed:
+    - ember-cli, with binary `ember`:
+	    - v3.10.0 on Node v12.2.0 with built-in npm (default)
+	    - v3.8.2 on Node v12.2.0 with built-in npm
+    - typescript, with binaries `tsc`, `tsserver`:
+	    - v3.4.5 on Node v12.2.0 with built-in npm
+	    - v3.0.3 on Node v12.2.0 with built-in npm (default)
+    - create-react-app, with binary `create-react-app`: v3.0.1 on Node v12.2.0 with built-in npm (default)
+    - yarn-deduplicate, with binary `yarn-deduplicate`: v1.1.1 on Node v12.2.0 with built-in npm (notice that this is not a *default*; assume the user ran `volta fetch` )
+
+They also have two projects with the following pins:
+
+- `~/node-only/package.json`:
+
+    ```json
+    {
+      "volta": {
+        "node": "v8.16.0",
+      }
+    }
+    ```
+
+- `~/node-and-yarn/package.json`:
+
+    ```json
+    {
+      "volta": {
+        "node": "v12.2.0",
+        "yarn": "v1.16.0"
+      }
+    }
+    ```
+
+### `volta list` (no flags)
+
+#### Human
+
+The format is:
+
+```sh
+$ volta list --human
```

Comment:
Ah, yes, this was a case where I forgot to update the rest of the text after realizing that it's much easier to do `--print=<more modes>`, and concluding it's probably better—because I don't think it's something you'll type often at all. I'll update the rest of the RFC text accordingly.